Version 3.0.2 throws error during start

Running my application with 3.0.2., I get the following error:

```
Error: Cannot find module './format'
Require stack:
- C:\apppath\node_modules\@e-invoice-eu\core\dist\e-invoice-eu.cjs.js
...
- C:\apppath\main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1151:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:120281)
    at Module._load (node:internal/modules/cjs/loader:992:27)
    at c._load (node:electron/js2c/node_init:2:13672)
    at Module.require (node:internal/modules/cjs/loader:1242:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (C:\appath\@e-invoice-eu\core\dist\e-invoice-eu.cjs.js:5:22)
    at Module._compile (node:internal/modules/cjs/loader:1391:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1451:10)
    at Module.load (node:internal/modules/cjs/loader:1214:32)
```

version 3.0.1 works just fine

--- a/.changeset/bumpy-impalas-rhyme.md
+++ b/.changeset/bumpy-impalas-rhyme.md
@@ -1,0 +1,8 @@
+---
+'@e-invoice-eu/core': patch
+---
+
+Move module type back to esnext.
+
+Setting `module` and `moduleResolution` to NodeNext broke the build
+(see #489).

--- a/.trivyignore
+++ b/.trivyignore
@@ -15,3 +15,8 @@ CVE-2026-31802
 # Date: 2026-04-02
 # Vulnerability in picomatch. This is part of npm but npm is not used.
 CVE-2026-33671
+
+# Date: 2026-04-04
+# Vulnerability in libinput-libs. This seems to be a false positive.
+# The library is not present in the image.
+CVE-2026-35093

--- a/packages/core/tsconfig.build.json
+++ b/packages/core/tsconfig.build.json
@@ -1,8 +1,7 @@
 {
 	"extends": "./tsconfig.json",
 	"compilerOptions": {
-		"module": "nodenext",
-		"moduleResolution": "nodenext",
+		"module": "esnext",
 		"typeRoots": ["node_modules/@types"],
 		"rootDir": "./src"
 	}

--- a/packages/core/tsconfig.build.json
+++ b/packages/core/tsconfig.build.json
@@ -2,6 +2,7 @@
 	"extends": "./tsconfig.json",
 	"compilerOptions": {
 		"module": "esnext",
+		"moduleResolution": "bundler",
 		"typeRoots": ["node_modules/@types"],
 		"rootDir": "./src"
 	}

